src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx

```diff
@@ -146,7 +146,14 @@ export default function WebexMeetingControlBar({
     ),
   );
 
-  const toggleCollapsed = () => setCollapsedShown(!collapsedShown);
+  const [withKey, setWithKey] = useState(false);
+
+  const toggleCollapsed = (withKeyboard) => {
```

Comment:
let's put everything in a single state:

```js
const toggleCollapsed = (withKey) => {
  setCollapsedShown((shown) => shown ? undefined : {withKey});
};
```

this means `collapsedShown` will be either `undefined` (collapsed controls not shown) or an object with the property `withKey` set to true or false.